sdk/ml/azure-ai-ml/tests/dsl/e2etests/test_dsl_pipeline_samples.py

```diff
@@ -348,3 +348,52 @@ def test_spark_job_with_builder_in_pipeline_without_entry(self, client: MLClient
         assert validation_result.error_messages == {
             "jobs.add_greeting_column.component.entry": 'Missing data for required field.',
         }
+
+    @pytest.mark.skipif(condition=is_live(), reason="need worskspace with datafactory compute")
+    @pytest.mark.e2etest
+    def test_data_transfer_copy_2urifolder_job_in_pipeline(self, client: MLClient) -> None:
+        from test_configs.dsl_pipeline.data_transfer_job_in_pipeline.copy_data.pipeline import (
+            generate_dsl_pipeline_from_yaml as data_transfer_job_in_pipeline,
+        )
+
+        pipeline = data_transfer_job_in_pipeline()
+        assert_job_cancel(pipeline, client)
+
+    @pytest.mark.skipif(condition=is_live(), reason="need worskspace with datafactory compute")
+    @pytest.mark.e2etest
+    def test_data_transfer_copy_2urifolder_job_with_builder_in_pipeline(self, client: MLClient) -> None:
+        from test_configs.dsl_pipeline.data_transfer_job_in_pipeline.copy_data.pipeline import (
+            generate_dsl_pipeline_from_builder as data_transfer_job_in_pipeline,
+        )
+        pipeline = data_transfer_job_in_pipeline()
+        assert_job_cancel(pipeline, client)
+
+    @pytest.mark.skipif(condition=is_live(), reason="need worskspace with datafactory compute")
+    @pytest.mark.e2etest
+    def test_data_transfer_copy_mixtype_job_in_pipeline(self, client: MLClient) -> None:
+        from test_configs.dsl_pipeline.data_transfer_job_in_pipeline.copy_data.pipeline import (
+            generate_dsl_pipeline_copy_mixtype_from_yaml as data_transfer_job_in_pipeline,
+        )
+
+        pipeline = data_transfer_job_in_pipeline()
+        assert_job_cancel(pipeline, client)
+
+    @pytest.mark.skipif(condition=is_live(), reason="need worskspace with datafactory compute")
+    @pytest.mark.e2etest
+    def test_data_transfer_copy_urifile_job_in_pipeline(self, client: MLClient) -> None:
+        from test_configs.dsl_pipeline.data_transfer_job_in_pipeline.copy_data.pipeline import (
+            generate_dsl_pipeline_copy_urifile_from_yaml as data_transfer_job_in_pipeline,
+        )
+
+        pipeline = data_transfer_job_in_pipeline()
+        assert_job_cancel(pipeline, client)
+
+    @pytest.mark.skipif(condition=is_live(), reason="need worskspace with datafactory compute")
```

Comment:
we might use serverless adf for our test, not sure if that's ready now

Reply:
@wangchao1230, how to use serverless adf in client? It seems there're no introduction on this

Reply:
just do: 
compute: azureml:serverless